tools/gfx/vulkan/render-vk.cpp

```diff
@@ -561,6 +561,7 @@ class VKDevice : public RendererBase
         VkAttachmentReference m_depthReference;
         bool m_hasDepthStencilAttachment;
         uint32_t m_renderTargetCount;
+        VkSampleCountFlagBits sampleCount = VK_SAMPLE_COUNT_1_BIT;
```

Comment:
m_sampleCount

Reply:
I've made this change locally, will be in the next PR I open.